package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "keep3r-v1-deploy",
```

```suggestion
  "name": "keep3r-v1",
```
No need of the deploy tag ? 😋